model/location.rb

```diff
@@ -18,3 +18,11 @@ class Location < Sequel::Model
 #  location_pkey | PRIMARY KEY btree (id)
 # Foreign key constraints:
 #  location_provider_fkey | (provider) REFERENCES provider(name)
+# Referenced By:
```

Comment:
Model annotation update should be included with the related migration commit.  A PR with just the migration commit should be able to pass CI.